django_redis/cache.py

```diff
@@ -72,7 +72,7 @@ def get(self, key, default=None, version=None, client=None):
             return self.client.get(key, default=default, version=version,
                                    client=client)
         except ConnectionInterrupted:
-            if DJANGO_REDIS_IGNORE_EXCEPTIONS:
+            if self._ignore_exceptions:
```

Comment:
the value of DJANGO_REDIS_IGNORE_EXCEPTIONS can be different from self._ignore_exceptions